cloudpickle/cloudpickle.py

```diff
@@ -384,7 +384,7 @@ def _save_subimports(self, code, top_level_dependencies):
                 # check if the package has any currently loaded sub-imports
                 prefix = x.__name__ + '.'
                 for name, module in sys.modules.items():
-                    if name.startswith(prefix):
```

Comment:
Would be great to have a comment here about why we check on the `name` being `None`.